concordia/admin/actions.py

```diff
@@ -1,9 +1,25 @@
+import uuid
+
 from django.contrib import messages
 from django.utils.timezone import now
 
 from ..models import Asset, Transcription, TranscriptionStatus
 
 
+def anonymize_action(modeladmin, request, queryset):
+    count = queryset.count()
+    for user_account in queryset:
+        user_account.username = uuid.uuid4()
+        user_account.email = ""
+        user_account.password = ""
+        user_account.save()
+
+    messages.info(request, f"Anonymized {count} user accounts")
+
+
+anonymize_action.short_description = "Anonymize user accounts"
```

Comment:
This might want to be a bit stronger about completely disabling the account as well.